.vib/influxdb/goss/goss.yaml

```diff
@@ -0,0 +1,28 @@
+file:
+  /bitnami/influxdb:
+    exists: true
+    filetype: directory
+    mode: "2775"
+    owner: root
+  /var/run/secrets/kubernetes.io/serviceaccount:
+    exists: {{ and .Vars.serviceAccount.create .Vars.serviceAccount.automountServiceAccountToken }}
```

Comment:
Personally, I think using `serviceAccount.automountServiceAccountToken` should be enough here. That param is the one that mounts the tested folder and it would also slim the `vars.yaml` file as well. No strong thoughts about this though.